dts/bindings/qspi/nxp,s32-qspi-sfp-frad.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP S32 Quad Serial Peripheral Interface (QSPI) Secure Flash Protection SFP FRAD.
+
+  The SFP FRAD performs second-level checks on input flash write and erase transactions,
+  based on the address range of each transaction.
+
+compatible: "nxp,s32-qspi-sfp-frad"
+
+include: base.yaml
+
+properties:
+  start-address:
+    type: int
+    default: 0
+    description: |
+      The flash memory region start address (around 64 KB boundary), default to 0.
```

Comment:
- Pls remove default value in the description for all properties (as `default:` feature from dts already describes this)
- Pls also explain why the default value is chosen

Reply:
If possible, pls add explanation in devicetree bindings, not via PR's comment.
`Pls remove default value in the description for all properties`: what i meant is removing `default to 0` in `The flash memory region start address (around 64 KB boundary), default to 0` (also for all places) because you already use `default:` feature to describe default value

Reply:
I removed the description mentions about the default value